create-elastic-index.js

```diff
@@ -1,16 +1,19 @@
 const esUtils = require('discovery-api-indexer/lib')
 const { decryptElasticCreds } = require('discovery-api-indexer/lib/kms-helper')
 const aws = require('aws-sdk')
+const dotenv = require('dotenv')
+
+dotenv.config({ path: './.env' })
 
 const init = async () => {
   aws.config.credentials = new aws.SharedIniFileCredentials({profile: "nypl-digital-dev"})
   const esUri = await decryptElasticCreds()
   esUtils.setConnection(esUri)
   // Create new index. Second param, deleteIfExists, ensures that the new index will overwrite
   // and old one with the same name.
-  await esUtils.resources.prepare('test-index-v1', true)
-  await esUtils.resources.prepare('test-index-v2', true)
-  console.log('Dropped and recreated test-index-v1 and v2')
+  await esUtils.resources.prepare(process.env['TEST_INDEX'] + 'v1', true)
+  await esUtils.resources.prepare(process.env['TEST_INDEX'] + 'v2', true)
```

Comment:
Here you're creating indexes called `test-index-vkv1` but in the cleanup script you're removing indexes like `test-index-vk-v1`